### PR TITLE
Dress Up updated to work with non-framework radios. 

### PR DESCRIPTION
**Pull request by kalebruscitti**
The changes made are as we discussed in #mm-scripts:
1. Simplify createDressUp to simply replace all acre radios with their base class.
2. Modify initPlayer to first give framework radios and then save the loadout, again replacing acre radios with their base class.
3. Remove radio adding from initPlayerRadios (as we don't want to add radios again on respawn anymore)
4. Add a case to onPlayerRespawn to re-add framework radios if the save-and-restore function is disabled (because Vanilla restore doesn't restore framework radios)

Finally, I also added dress up information to the readme file.

### CODE REVIEW

File: tcs/fn_initPlayer.sqf [hunk 1]
```diff
@@ -19,11 +19,15 @@ if (TCS_var_addSurgicalKitsToMedics) then {
 	};
 };
 
-if (TCS_var_saveAndRestoreLoadouts) then {
-	TCS_var_playerLoadout = getUnitLoadout player;
+// Add any radios from the framework, and wait until that is done.
```
Comment (gkovalechyn) [inline tcs/fn_initPlayer.sqf]: I don't think this comment is necessary

File: tcs/base/fn_onPlayerRespawn.sqf [hunk 2]
```diff
@@ -9,6 +9,14 @@ if (TCS_var_saveAndRestoreLoadouts) then {
...
 };
 
+/** 
+	In my testing, this specific case ended up with the player missing framework radios on respawn,
+	thus I add this line of code to add them back.
+*/
+if ((TCS_var_addRadiosToPlayers) and (not TCS_var_saveAndRestoreLoadouts)) then {
+	[] call TCS_fnc_addConfigurationRadiosToPlayer;
+};
+
```
Comment (gkovalechyn) [inline tcs/base/fn_onPlayerRespawn.sqf:19]: I'm not sure about this behavior, what do we expect of the MM in this case? He doesn't want the loadouts saved but wants the radios on respawn? Does that make sense?

Comment (YouGotSchott) [reply tcs/base/fn_onPlayerRespawn.sqf:19]: This could be useful for test map Zeus missions. We can discuss it more in #mm-scripts

Comment (kalebruscitti) [reply tcs/base/fn_onPlayerRespawn.sqf:19]: Consider: The MM wants a mission where your loadouts are not reset on death, so when you respawn you have whatever you died with. Thus they set saveAndRestore false. Then they use framework to give radios. The natural expected behaviour is that if you die with a (say) 343, then you respawn with the 343. I tested this case and I lost all my framework radios on respawn, so I added these lines to add them back.

File: tcs/fn_initPlayer.sqf (hunk 1 above)
Comment (kalebruscitti) [reply tcs/fn_initPlayer.sqf]: I removed it.

**Comment by gkovalechyn** [comment]
I have no idea why the check is saying that the changelog was modified but could you add what was changed to the changelog?

**Comment by kalebruscitti** [comment]
@gkovalechyn  I wrote the changelog as requested

File: CHANGELOG.md [hunk 3]
```diff
@@ -12,11 +12,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Headless AI forced stances fixed, performance improvements, and new UGL/AT setting.
 * Headless state machines updates.
 * Updated fn_adminInit.sqf with new Mods and Admins.
+* Re-ordered initPlayer to give framework radios before saving loadout, and to replace ACRE radios with their base classes.
 
 ### Fixed
 * Master PR validation script paramters.
 * Changed files not working properly for pull requests.
+* Moved the VehicleRespawn folder to fix the release build
```
Comment (YouGotSchott) [inline CHANGELOG.md:20]: This line got duplicated somehow; could we remove that?

**Review by YouGotSchott — CHANGES_REQUESTED** [review]

**Review by gkovalechyn — APPROVED** [review]

**Review by YouGotSchott — APPROVED** [review]

**Pull request merged**